ApiExtractor/fileout.cpp

```diff
@@ -206,7 +206,6 @@ bool FileOut::done()
                                        .arg(fileWrite.fileName()));
                 return false;
             }
-            QTextCodec::setCodecForCStrings(QTextCodec::codecForName("UTF-8"));
```

Comment:
Be *very careful* if you remove this that you make sure every call to `QString::toAscii` that could be called after this executes is replaced with `toUtf8` *and* ***not*** `toLatin1`, or you will break things massively in the face of non-ASCII strings.